client/lib/plans/constants.js

```diff
@@ -374,7 +374,7 @@ const getPlanBusinessDetails = () => ( {
 			);
 		}
 		return i18n.translate(
-			'{{strong}}Best for Small Business:{{/strong}} Power your' +
+			'{{strong}}Best for Small Businesses:{{/strong}} Power your' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( '{{strong}}Best for Small Business:{{/strong}} Power your business website with unlimited premium and business theme templates, Google Analytics support, unlimited storage, and the ability to remove WordPress.com branding.' )` *ES Score: 19*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plans-features)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).